pagopa/io-app
# feat: [IOPID-296] Update the generate nonce and set it as mandatory

## Short description
This PR aligns the get nonce to the [new specification](https://github.com/pagopa/io-backend/pull/1070) and bump the backend version from `13.19.1` to `13.20.0`.

## How to test
Test a login flow with Fast Login enabled and ensure that all works fine. Use this [io-dev-api-server PR](https://github.com/pagopa/io-dev-api-server/pull/315) to test the login flow.

<details><summary>Example</summary>
<p>

<video src="https://github.com/pagopa/io-app/assets/16268789/a9345b74-b3d8-4f28-9e5d-f7508dc0e269" />

</p>
</details>
